src/aws/secrets_retriever.py

```diff
@@ -0,0 +1,17 @@
+import boto3
+
+from settings.secrets.retrievers.base_retriever import BaseSecretsRetriever
+
+
+class SSMSecretsRetriever(BaseSecretsRetriever):
+    def __init__(self):
+        region_name = "eu-central-1"
+        self._common_prefix = "/BlackSheepLearns/dev/"
```

Comment:
This aches me. Maybe it should be injectable? For now it'll do, but in the future this might require refactoring.

Reply:
Decision: postpone the adjustment until it's actually useful. For now I'm only using 1 environment, so there's no point in adding complexity.